Moda.Web/src/moda.web.reactclient/src/app/components/common/planning/program-increment-objectives-grid.tsx

```diff
@@ -201,16 +214,16 @@ const ProgramIncrementObjectivesGrid = ({
   ]
 
   const onCreateObjectiveFormClosed = (wasCreated: boolean) => {
-    setOpenCreateObjectiveModal(false)
+    setOpenCreateObjectiveForm(false)
     if (wasCreated) {
-      loadObjectives()
+      refresh()
```

Comment:
Since this is using the objectivesQuery, if the mutation is used to create or update an objective, won't the data be invalidated and refetched automatically?

Reply:
Good call